inc/bill.class.php

```diff
@@ -200,11 +200,13 @@ public function rawSearchOptions()
 
         $tab[] = [
             'id'            => 8,
-            'table'         => PluginOrderOrder::getTable(),
-            'field'         => 'name',
-            'name'          => __('Order', 'order'),
-            'datatype'      => 'itemlink',
-            'itemlink_type' => 'PluginOrderOrder',
+            'table'	    => 'glpi_plugin_order_bills',
+	    'field' 	    => 'order_name',
+	    'name'	    => __('Order', 'order'),
+	    'datatype' 	    => 'string',
+	    'nosearch' 	    => true,
+	    'massiveaction' => false,
+	    'computation'   => '(SELECT name FROM glpi_plugin_order_orders WHERE glpi_plugin_order_orders.id = glpi_plugin_order_bills.plugin_order_orders_id)',
```

Comment:
This should allow searching to list all bills associated with a specific order.

There is no need to use `computation`; refer to the previous search option that uses the 'dropdown' data type.